Calculator.py

```diff
@@ -17,6 +17,9 @@ def run_operation(number_1,  number_2, operation):
     elif operation == '/':
         print('{} / {} = '.format(number_1, number_2), end = '')
         return number_1 / number_2
+    elif operation == '%':
+        print('{} % {} = '.format(number_1, number_2), end = '')
+        return number_1 % number_2
     else:
         raise Exception('This operator is invalid. Try running the program again.')
```

Comment:
Ideally, the user should be able to try again without running the program. I think by checking in the gather_inputs() for valid entries, you should be able to go through the run_operation without any issues.

Reply:
> Ideally, the user should be able to try again without running the program. I think by checking in the gather_inputs() for valid entries, you should be able to go through the run_operation without any issues.

I think this is a great idea- maybe adding some try except blocks in the gather_inputs() functions, along with placing that whole piece in the while loop (while any of the inputs are invalid), any of the direct input errors could be handled.

Reply:
I agree that preventing the program from crashing due to incorrect input would be better!

Reply:
This is a great idea! I think there could be a few more error handling blocks of code added to catch incorrect inputs.

Reply:
Yes! Using a try/except will help to do this.

Reply:
> Ideally, the user should be able to try again without running the program. I think by checking in the gather_inputs() for valid entries, you should be able to go through the run_operation without any issues.

I think this is the right step forward. It's good that our program doesn't do anything incorrect when an invalid operator is inputted but to better the UX, we should maybe add a print statement saying this is an invalid operation and allowing the user to immediately input another operation.